toDoApp/src/store/todo/actions.tsx

```diff
@@ -1,14 +1,68 @@
-import { ActionType, TodoState } from './types'
+import { createAsyncThunk } from '@reduxjs/toolkit'
+import { TodoDraft, TodoState } from './types'
+import axios from 'axios'
+import { goBack } from '../../common/rootNavigation'
+import { ListItemType } from '../../screens/Home/types'
 
 export const clear = (state: TodoState) => {
-  state.value = state.value.filter((item) => !item.checked)
+  state.value = state.value.filter((item) => !item.completed)
 }
 
-export const check = (state: TodoState, action: ActionType) => {
-  const index = state.value.findIndex((elem) => action.payload === elem.id)
-  state.value[index].checked = !state.value[index].checked
-}
+export const getTodos = createAsyncThunk<ListItemType[], {}>(
+  'todos/getTodos',
+  async () => {
+    const result = await axios.get(
+      'https://rn-training-backend.herokuapp.com/todos',
```

Comment:
This is an overkill improvement but it would be nice to read about the environment variables and how to use it. Because right now you are exposing this to the public and its not a good practice.
Check this [library](https://github.com/luggit/react-native-config)